dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/WorkflowVersionDAO.java

```diff
@@ -63,4 +93,45 @@ public List<WorkflowVersion> getTagsByWorkflowIdOrderedByLastModified(long workf
         query.setMaxResults(limit);
         return query.getResultList();
     }
+
+    private List<Predicate> processQuery(String sortCol, String sortOrder, CriteriaBuilder cb, CriteriaQuery query, Root<WorkflowVersion> version) {
+        List<Predicate> predicates = new ArrayList<>();
+
+        if (!Strings.isNullOrEmpty(sortCol)) {
```

Comment:
If `sortCol` is null or empty, you should have an `else` block that sorts by id. Postgres queries don't have a default sort order, so there's no guarantee you'll get the same results for the same query; the user might end up with duplicates or skipped entries when paginating through versions.